channel/src/main/java/com/synopsys/integration/alert/channel/jira2/common/delegate/JiraIssueCommenter.java

```diff
@@ -17,7 +17,7 @@
 import com.synopsys.integration.jira.common.model.request.IssueCommentRequestModel;
 
 public abstract class JiraIssueCommenter extends IssueTrackerIssueCommenter<String> {
-    private final ThrowingConsumer<IssueCommentRequestModel, IntegrationException> addCommentConsumer;
+    private final ThrowingConsumer<IssueCommentRequestModel, IntegrationException> addCommentConsumer; // TODO: Just expose another abstract method instead of this
```

Comment:
Will this be a follow-up PR?

Reply:
Alternatively, does this class need to be abstract? Could the necessary fields be passed through the constructor such that no subclasses would be needed?

Reply:
This will be a follow-up PR. As the code is now, the class probably doesn't need to be abstract. But I prefer adding an abstract method instead of the field being passed into the constructor. We can discuss in the next PR.